frontend/src/app/pages/exercises/exercise/shared/exercise-map/feature-managers/vehicle-feature-manager.ts

```diff
@@ -174,4 +188,99 @@ export class VehicleFeatureManager extends MoveableFeatureManager<Vehicle> {
+        const text = `${filled}/${vehicle.patientCapacity}`;
+
+        // Farben bestimmen (optional nach kritischster SK)
+        let backgroundColor = 'rgba(255, 255, 255, 0.85)';
+        let backgroundStroke = 'rgb(255, 255, 255)';
+        let color = 'black';
+
+        if (config.vehicleStatusInSkColor && filled > 0) {
+            const state = selectStateSnapshot(selectExerciseState, this.store);
+            const patients = Object.keys(vehicle.patientIds)
+                .map((id) => state.patients[id])
+                .filter(Boolean);
+
+            // Sichtbarer Status je Konfiguration
+            const getStatus = (p: any) =>
+                config.pretriageEnabled ? p.pretriageStatus : p.realStatus;
+
+            let sk: 'black' | 'blue' | 'green' | 'red' | 'yellow' | undefined;
+            if (patients.some((p) => p && getStatus(p) === 'red')) sk = 'red';
+            else if (patients.some((p) => p && getStatus(p) === 'yellow'))
+                sk = 'yellow';
+            else if (patients.some((p) => p && getStatus(p) === 'green'))
+                sk = 'green';
+            else if (patients.some((p) => p && getStatus(p) === 'blue'))
+                sk = 'blue';
+            else if (patients.some((p) => p && getStatus(p) === 'black'))
+                sk = 'black';
+
+            if (sk === 'red') {
+                backgroundColor = 'rgba(220, 53, 69, 0.85)';
+                backgroundStroke = 'rgb(220, 53, 69)';
+                color = 'white';
+            } else if (sk === 'yellow') {
+                backgroundColor = 'rgba(255, 193, 7, 0.85)';
+                backgroundStroke = 'rgb(255, 193, 7)';
+                color = 'black';
+            } else if (sk === 'green') {
+                backgroundColor = 'rgba(40, 167, 69, 0.85)';
+                backgroundStroke = 'rgb(40, 167, 69)';
+                color = 'white';
+            } else if (sk === 'black') {
+                backgroundColor = 'rgba(15, 15, 15, 0.85)';
+                backgroundStroke = 'rgb(15, 15, 15)';
+                color = 'white';
+            } else if (sk === 'blue') {
+                backgroundColor = 'rgba(0, 123, 255, 0.85)';
+                backgroundStroke = 'rgb(0, 123, 255)';
+                color = 'white';
+            }
+        }
+
+        // Text-Style über dem Fahrzeug rendern
+        const resolution = this.olMap.getView().getResolution() ?? 1;
+        const scale = 1 / resolution;
+        const fontPx = 2 * scale;
+
+        const textStyle = new Style({
+            text: new OlText({
+                text,
+                font: `${fontPx}px sans-serif`,
+                fill: new Fill({ color }),
+                backgroundFill: new Fill({ color: backgroundColor }),
+                backgroundStroke: new Stroke({
+                    color: backgroundStroke,
+                    width: 0.5 * scale,
+                }),
+                offsetY: -6 * scale,
```

Comment:
I'd prefer the status display to be a bit closer to the vehicle, for example `-5 * scale`